clang/lib/Parse/ParseStmt.cpp

```diff
@@ -1553,6 +1553,13 @@ StmtResult Parser::ParseIfStatement(SourceLocation *TrailingElseLoc) {
       IsConsteval = true;
       ConstevalLoc = ConsumeToken();
     }
+
+    if (Tok.is(tok::code_completion)) {
```

Comment:
This needs to go in an `else` after the `Tok.is(tok::kw_consteval)`. Otherwise, if you invoke completion **after** `consteval`, e.g.:

`if consteval ^` <-- here

then you incorrectly get `constexpr` and `consteval` suggested.